tools/shared/pragma.c

```diff
@@ -271,6 +271,63 @@ static void lcase(char *);
 static bool craydir; /* true if cray directive */
 static bool sundir;  /* true if sun directive */
 
+/*
+ * reads vectorlength directive () set proper flags or raise error)
+ */
+static void scan_vectorlength(int* toSet,int* Num, int typ)
```

Comment:
`Num` is a simple poitner type, should be named `num`
Also, keep the project's formatting of pointer declaration, should be `int *num` not `int* num`